modules/circuits/views/nodes/nodes/nodes.php

```diff
@@ -90,12 +90,12 @@
     </select>
   </div>
 
-<!--
+
   <div class="form-group">
     <label for="exampleInputPassword1">Source VLAN</label>
     <input type="number" maxlength="4" class="form-control" id="source_vlan" name="source_vlan" placeholder="0-4096" >
```

Comment:
@usmanazFIU can you please change the placeholder here to inform VLAN id will be from 1-4095 ? (0 and 4096 should NOT be considered valid values according to the Datamodel spec https://docs.google.com/document/d/1lgxjIT144EFu1G_OVcU19hN1cSUT_v2-tE0Z-7UlkNg/edit#bookmark=id.vixmudxbm9j7)

Reply:
sure. let me change it